Packs/Keyfactor/Integrations/KeyfactorDev/KeyfactorDev.py

```diff
@@ -0,0 +1,361 @@
+            f" val.UpdateDate && val.UpdateDate == obj.UpdateDate)": entry_context
+        }
+        human_readable = tableToMarkdown(name=title,
+                                         t=human_readable_ec,
+                                         removeNull=True)
+        return human_readable, context_entry, raw_response
+    else:
+        return {}, {}, raw_response
+
+
+@logger
+def get_enrollment_csr_context_my_command(client: Client, *_) -> Tuple[object, dict, Union[List, Dict]]:
+    """Get all Enrollment CSR Context My (templates)
+
+    Args:
+        client: Client object with request
+
+    Returns:
+        human readable (markdown format), entry context and raw response
+    """
+
+    raw_response: dict = client.get_enrollment_csr_context_my()
+
+    if raw_response:
+        title = f'{INTEGRATION_NAME} - Get enrollment csr context my'
+        entry_context, human_readable_ec = get_enrollment_csr_context_my_ec(raw_response)
+        context_entry: Dict = {
+            f"{INTEGRATION_CONTEXT_NAME}.CSRTemplate.Lists(val.UniqueID && val.UniqueID == obj.UniqueID &&"
+            f" val.UpdateDate && val.UpdateDate == obj.UpdateDate)": entry_context
+        }
+        human_readable = tableToMarkdown(name=title,
+                                         t=human_readable_ec,
+                                         removeNull=True)
+        return human_readable, context_entry, raw_response
+    else:
+        return f'{INTEGRATION_NAME} - Could not find any results for given query', {}, {}
+
+
+@logger
+def test_module_command(client: Client, *_) -> Tuple[None, None, str]:
+    """Performs a basic GET request to check if the API is reachable and authentication is successful.
+
+    Args:
+        client: Client object with request
+        *_: Usually demisto.args()
+
+    Returns:
+        'ok' if test successful.
+
+    Raises:
+        DemistoException: If test failed.
+    """
+    results = client.test_module()
+    if 'Templates' in results or 'CertificateInformation' in results:
+        return None, None, 'ok'
+    raise DemistoException(f'Test module failed, {results}')
+
+
+def main():
+    params = demisto.params()
+    verify_ssl = not params.get('insecure', False)
+    proxy = params.get('proxy')
+    username = params.get('username')
+    password = params.get('password')
+    client = Client(
+        base_url=params.get('host'),
+        verify=verify_ssl,
+        proxy=proxy,
+        auth=(
+            username, password
+        )
+    )
+    command = demisto.command()
+    commands = {
+        'test-module': test_module_command,
+        f'{INTEGRATION_COMMAND_NAME}-get-enrollment-csr-context-my': get_enrollment_csr_context_my_command,
+        f'{INTEGRATION_COMMAND_NAME}-post-enrollment-csr': post_enrollment_csr_command
+    }
+
+    try:
+        readable_output, outputs, raw_response = commands[command](client=client, **demisto.args())
+        return_outputs(readable_output, outputs, raw_response)
```

Comment:
`return_outputs` is deprecated, please use `CommandResults` and `return_results`.